### PR TITLE
OKTA-623282 Add scope claim to Token Inline Hook reserved list

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Added `scp` claim to token hook reserved list
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-623282](https://oktainc.atlassian.net/browse/OKTA-623282)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/token-hook/index.md](https://github.com/okta/okta-developer-docs/blob/42de35a85a8920146e986f022a7bec93b3018d77/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Ftoken-hook%2Findex.md) | 81 | 85 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/586c87fd-284e-4258-ab95-5f6494b3d293) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.cae0a6e3-b5c3-4cc3-a53e-314bade1e5e7?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/token-hook/index.md](https://github.com/okta/okta-developer-docs/blob/a419a0105f54ba03387132a10e8dbad28a58d815/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Ftoken-hook%2Findex.md) | 81 | 86 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e838bcbf-266c-45cc-a121-e94818654e7a) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.c90348bc-305f-4b0f-a4a9-e20db31bd634?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/token-hook/index.md](https://github.com/okta/okta-developer-docs/blob/b83fb31010f36fa0cba5de015801d094eab3fed6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Ftoken-hook%2Findex.md) | 81 | 86 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/664edd08-92c6-4d24-806d-1a9082bd1590) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.e26f3a8a-b58a-4d83-9057-96fb8df87751?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/reference/token-hook/index.md [hunk 1]
```diff
@@ -126,15 +126,15 @@ Okta defines various reserved claims that can't be overridden. When you add a cu
 
 | Claim name     | Token type        |
 |----------------|-------------------|
-| acr            | access token      |
-| amr            | access token      |
+| authorization_details | access token      |
 | as_uri         | access token      |
-| cid            | access token      |
-| groups         | access token      |
+| cnf (if [DPoP](/docs/guides/dpop/main) enabled) | access token      |
```
Comment (dannavarro-okta) [inline packages/@okta/vuepress-site/docs/reference/token-hook/index.md:131]: I will leave this up to a Doc reviewer to decide if this is how we want to doc it. Just pointing out that this claim is already present for ID token as well. https://github.com/okta/okta-developer-docs/pull/4250/files#diff-f2d1b4f12feefd6381dc18b2d147e1f3a34a2817818a7f0e0ee4751c5706f363L150

**Review by dannavarro-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/reference/token-hook/index.md (hunk 1 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/reference/token-hook/index.md:131]: Yes, I left `cnf` in the "ID token" section that follows. But I could also add to the "both" portion of the table with the DPoP caveat.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/token-hook/index.md](https://github.com/okta/okta-developer-docs/blob/d05914df42f5d7ffff65e7c1c2f8fd49966a7ee3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Ftoken-hook%2Findex.md) | 80 | 87 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/777c8f61-29fb-408a-9ea2-9678e147d8d8) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.3c520a70-7c66-4b83-a45e-78318cbd1ecb?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Review by isaacdurham-okta — APPROVED** [review]

**Pull request merged**